GeoLib/GEOObjects.cpp

```diff
@@ -551,9 +551,83 @@ void GEOObjects::renameGeometry(std::string const& old_name,
     }
 }
 
-const GeoLib::GeoObject* GEOObjects::getGeoObject(const std::string &geo_name,
-                                                            GeoLib::GEOTYPE type,
-                                                            const std::string &geo_obj_name) const
+void GEOObjects::markPointsAsUsed(std::string const& geo_name,
+                                  std::vector<bool>& flags)
+{
+    GeoLib::PolylineVec const* const ply_obj(getPolylineVecObj(geo_name));
+    if (ply_obj)
+    {
+        std::vector<GeoLib::Polyline*> const& lines(*ply_obj->getVector());
+        for (auto line : lines)
```

Comment:
Minor: better to write `auto* line` forcing the value to be a pointer. Same for surfaces.

Reply:
:heavy_check_mark: